test/decode.js

```diff
@@ -0,0 +1,18 @@
+import chai, { expect } from 'chai'
```

Comment:
for testing encode and decode, theres a way to do a integrative test that tests encode and decode at the same time. im not sure if yall have started encode, just a tip :)